src/main/drivers/dshot.h

```diff
@@ -120,6 +120,8 @@ void updateDshotTelemetryQuality(dshotTelemetryQuality_t *qualityStats, bool pac
 void initDshotTelemetry(const timeUs_t looptimeUs);
 void updateDshotTelemetry(void);
 
+void initEscSensor(void);
```

Comment:
The ESC has nothing to do with Dshot. Take a look at `getEscSensorData()` in `esc_sensor.c`

Reply:
Thanks will refactor a bit as it would be used by analog protocols also?

Reply:
@haslinghuis - As per Karatebrot, this line shouldn’t be needed here?